README.md

```diff
@@ -72,3 +72,7 @@ The following options are available.
  * `--lsm_port` port to use to ssh to the remote orchestrator, overrides INMANTA_LSM_PORT
  * `--lsm_environment` the environment to use on the remote server (it is created if it doesn't exist), overrides INMANTA_LSM_ENVIRONMENT
  * `--lsm_noclean` Don't cleanup the orchestrator after last test (for debugging purposes)
+ * `--lsm_ssl` Connect to the remote orchestrator using SSL/TLS
```

Comment:
We also need to add some explanation saying how this option should be used. Intuitively I thought this was a flag, but this option has a value as well. We should indicate which values are valid.